### ISSUE TITLE
Working with Async code

### ISSUE THREAD

**Issue by stzaske** [Microsoft/WinAppDriver#645]
I'm trying to learn WinAppDriver and looking over your Bing examples it looks like you're just waiting a certain amount of time.  Is there no better option for Async code? In my UWP app I'm pulling down API code and want showing the results in a grid.  How can I instruct WinApDriver to wait until the results return?

**Comment by naeemakram** [comment]
Maybe you could implement a circular wait based on some properties of the grid.

**Comment by PandaMagnus** [comment]
Agreed, a better option would be to use something like WebDriverWait, and poll for something like the Bing pane being visible or enabled.
